scripts/Muon/GUI/Common/plot_widget/raw_pane/raw_pane_presenter.py

```diff
@@ -21,11 +21,21 @@ def __init__(self, view, model, context,figure_presenter):
         self._view.hide_plot_raw()
         self._view.disable_tile_plotting_options()
         self._view.set_is_tiled_plot(True)
-        self.data_observer = GenericObserver(self.handle_data_updated)
+        self.new_data_observer = GenericObserver(self.handle_new_data)
```

Comment:
its only looking for new data loaded and not just the data has been updated. Since these are raw plots, the data only changes when new data is loaded.

Reply:
Now you've explained it the renaming makes sense!